docs/deployment/resource-providers/mesos.md

```diff
@@ -84,113 +77,127 @@ of failures.
 If you don't have a running DC/OS cluster, please follow the
 [instructions on how to install DC/OS on the official website](https://dcos.io/install/).
 
-Once you have a DC/OS cluster, you may install Flink through the DC/OS
-Universe. In the search prompt, just search for Flink. Alternatively, you can use the DC/OS CLI:
+Once you have a DC/OS cluster, you may install Flink through the DC/OS Universe. Just 
+search for Flink in the search prompt. Alternatively, you can use the DC/OS CLI:
 
-    dcos package install flink
+{% highlight bash %}
+dcos package install flink
+{% endhighlight %}
 
 Further information can be found in the
-[DC/OS examples documentation](https://github.com/dcos/examples/tree/master/1.8/flink).
-
-
-## Mesos without DC/OS
-
-You can also run Mesos without DC/OS.
-
-### Installing Mesos
-
-Please follow the [instructions on how to setup Mesos on the official website](http://mesos.apache.org/getting-started/).
-
-After installation you have to configure the set of master and agent nodes by creating the files `MESOS_HOME/etc/mesos/masters` and `MESOS_HOME/etc/mesos/slaves`.
-These files contain in each row a single hostname on which the respective component will be started (assuming SSH access to these nodes).
+[DC/OS examples documentation](https://github.com/dcos/examples/tree/master/flink).
```

Comment:
The documentation here installs Flink 1.3.1 (https://github.com/dcos/examples/tree/master/flink/1.10). Do we really want to promote this? If there is no newer DC/OS package, then I'd suggest to go for the option w/o DC/OS and the latest Flink version.

Reply:
I removed the DC/OS section and focused on deploying Flink without additional tools.